src/__tests__/test.js

```diff
@@ -1,12 +1,13 @@
 'use strict';
 
 var generate = require('../index');
+
 var options = {
     seed: 22,
     classes: [
         {
-            nbSample: 500,
-            elements: [
+            nbSpectra: 500,
+            units: [
```

Comment:
This array contain parameter of the distribution to be sampled, Are you agree if we change units per  componentsParameters?